### ISSUE TITLE
 Command `code` Not Working in Regular Terminal

### ISSUE THREAD

**Issue by ruslanguns** [LGUG2Z/nixos-wsl-starter#18]
I'm having trouble with the code command not being recognized when I try to run it from my regular terminal. Weirdly enough, it works just fine from the integrated terminal in VSCode, which defaults to sh. This seems like an environment setup issue.

```fish
nixos in ~/configuration on  master [!]
❯ code .
code: command not found
```
I'll attach a screenshot showing that the code command works in the VSCode terminal, and I'll also include the path to the code binary. Any help to sort this out would be really appreciated!

![image](https://github.com/user-attachments/assets/43a9afcd-60cc-4557-bb10-3156bbfcb9f3)

From integrated terminal: 

```fish
nixos in ~/configuration on  master [!] 
❯ which code
/home/rus/.vscode-server/bin/138f619c86f1199955d53b4166bef66ef252935c/bin/remote-cli/code
```

**Comment by ruslanguns** [comment]
I found a workaround to use the `code` command from the NixOS terminal by creating an alias for the fish shell:

```
code = "/mnt/c/Users/[my-user]/AppData/Local/Programs/'Microsoft VS Code'/bin/code";
```

This should work well for now. I'm also planning to modify VSCode to use fish as the default shell. Feel free to close this ticket, or if someone else wants to pick it up as a `Good First Issue`, that would be awesome!

I'll be submitting a PR with this snippet in `home.nix` in case it helps others.

**Comment by LGUG2Z** [comment]
Closed with #19 !
